File: tests/bench/LoopSum.java
// Same program as loop_sum.jai.
public class LoopSum {
    public static void main(String[] args) {
        long total = 0;
        for (long i = 0; i < 50000000L; i++) {
            total += i % 7;
        }
        System.out.println(total);
    }
}

File: tests/bench/loop_sum.cpp
// Same program as loop_sum.jai. See tests/bench/README.md for why the C++ and
// Java rows are here and what they are not.
#include <cstdint>
#include <cstdio>

int main() {
    int64_t total = 0;
    for (int64_t i = 0; i < 50000000; i++) {
        total += i % 7;
    }
    std::printf("%lld\n", (long long)total);
    return 0;
}
